lambdas/search/index.py

```diff
@@ -46,8 +44,10 @@ def lambda_handler(request):
     """
     Proxy the request to the elastic search.
     """
+
     action = request.args.get('action')
     indexes = request.args.get('index')
+    terminate_after = os.getenv('MAX_DOCUMENTS_PER_SHARD')
```

Comment:
This needs a default or else it can return `None`.
If there's no safe default (unlikely) then we should immediately throw an exception.
We should also make sure this number is an integer. The best way to do that is probably:
```
 locale.setlocale(locale.LC_ALL, 'en_US.UTF-8')
locale.atoi(terminate_after)
```
Docs are not really clear on kwarg type https://elasticsearch-py.readthedocs.io/en/master/api.html#elasticsearch.Elasticsearch.search